cni/network/multitenancy.go

```diff
@@ -203,7 +208,77 @@ func (m *Multitenancy) GetContainerNetworkConfiguration(
 	return ncResponse, hostSubnetPrefix, err
 }
 
-func (m *Multitenancy) getContainerNetworkConfigurationInternal(
+// get all network container configurations by given orchestratorContext
+func (m *Multitenancy) GetNetworkContainersWithOrchestratorContext(
+	ctx context.Context, nwCfg *cni.NetworkConfig, podName, podNamespace string,
+) (*[]cns.GetNetworkContainerResponse, []net.IPNet, error) {
+	var podNameWithoutSuffix string
+
+	if !nwCfg.EnableExactMatchForPodName {
+		podNameWithoutSuffix = network.GetPodNameWithoutSuffix(podName)
+	} else {
+		podNameWithoutSuffix = podName
+	}
+
+	log.Printf("Podname without suffix %v", podNameWithoutSuffix)
+
+	ncResponses, hostSubnetPrefix, err := m.getNetworkContainersWithOrchestratorContextInternal(ctx, podNamespace, podNameWithoutSuffix)
```

Comment:
should it be hostSubnetPrefixes since you are returning a slice?

Reply:
Yes, it should be named "hostSubnetPrefixes" since getNetworkContainersWithOrchestratorContextInternal returns a slice of hostSubPrefix